runipy/notebook_runner.py

```diff
@@ -208,6 +214,8 @@ def run_cell(self, cell):
 
                     data_out = data if not json_encode else json.dumps(data)
                     setattr(out, attr, data_out)
+                    if self.logback:
+                        print(data_out)
             elif msg_type == 'pyerr':
```

Comment:
I might have forgotten one spot where to place this additional `print`.

And it should probably be `logging.info` instead of `print`. What do you think?